core/src/org/pentaho/di/core/row/value/ValueMetaBase.java

```diff
@@ -578,6 +588,70 @@ public void setCaseInsensitive( boolean caseInsensitive ) {
     this.caseInsensitive = caseInsensitive;
   }
 
+   /**
+   * @return the collatorDisabled
+   */
+  @Override
+  public boolean isCollatorDisabled() {
+    return collatorDisabled;
+  }
+
+  /**
+   * @param collatorDisabled
+   *          the collatorDisabled to set
+   */
+  @Override
+  public void setCollatorDisabled( boolean collatorDisabled ) {
+    this.collatorDisabled = collatorDisabled;
+  }
+  
+  
+    /**
+   * @ sets the collator Locale
+   */
+  @Override
+  public void setCollatorLocale( Locale locale ) {
+    this.collator = Collator.getInstance( locale );
+  }
+
+    /**
+   * @get the collatorStrength
+   */
+  @Override
+  public String getCollatorStrength() {
+    return collatorStrength;
+  }
+
+  /**
+   * @param collatorStrength
+   *          the collatorStrength to set
+   */
+  @Override
+  public void setCollatorStrength( String collatorStrength ) {
```

Comment:
Should probably be an int which would eliminate the need to do the mapping this way. The only useful place for the string is in the UI classes.